css-nesting-1/Overview.bs

```diff
@@ -248,6 +262,12 @@ Direct Nesting {#direct}
 		}
 		/* Invalid because the second selector in the list doesn't
 		   contain a nesting selector. */
+
+		.foo {
+			color: red;
+			&&.bar { color: blue; }
+		}
+		/* Invalid because the 1st & isn't at the start of the selector */
```

Comment:
Do you mean the 2nd &?

Reply:
I didnt. Let me update the verbiage to be more clear 👍